images/ubuntu/templates/ubuntu-22.04-prod.pkr.hcl

```diff
@@ -54,7 +54,7 @@ variable "misc_script_folder" {
 }
 
 source "docker" "blacksmith" {
-  image  = "blacksmithcihello/rootfs-packer:150224-2"
+  image  = "blacksmithcihello/rootfs-packer:150224-3"
```

Comment:
is this correct?

Reply:
yeah this is the current tag on prod - https://github.com/FastActions/fa/blob/main/infra/ansible/pull_latest_rootfs_zfs.yaml#L13